src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -141,6 +146,12 @@ def migration_update_func(cmd, client, resource_group_name, server_name, migrati
         operationSpecified = True
         properties = "{\"properties\": {\"triggerCutover\": \"true\"} }"
 
+    if start_data_migration is True:
+        if operationSpecified is True:
+            raise MutuallyExclusiveArgumentError("Incorrect Usage: Can only specify one update operation.")
+        operationSpecified = True
+        properties = "{\"properties\": {\"startDataMigration\": \"true\"} }"
+
     if operationSpecified is False:
         raise RequiredArgumentMissingError("Incorrect Usage: Atleast one update operation needs to be specified.")
```

Comment:
```suggestion
        raise RequiredArgumentMissingError("Incorrect Usage: At least one update operation needs to be specified.")
```

Reply:
Here, we can specify only 1 update operation . with `At least` means , we can specify more than equal to one. Am i missing something else here?